### ISSUE TITLE
Dashboard: Empty header cells <th>

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#20]
URL: https://hcdev.openplus.ca/en/user/203 (Dashboard)
User Agent: Chrome, Firefox
Language: English

Issue Description: There are tables with empty header cells. The table under Alerts and recalls in progress has empty header cells first and last columns. th should not be empty.

Steps to reproduce: View source code

Expected results: Header cells shouldn't be empty and if these cannot have any content then they should be changed to td.

![image](https://user-images.githubusercontent.com/14280062/127520964-1bc2576d-4cd8-4073-8014-53a04f60a3f7.png)

![image](https://user-images.githubusercontent.com/14280062/127520893-f2746a9f-cebf-44f4-8c42-62462f4110a1.png)

**Comment by qanitah** [comment]
Fixed
